### DISCUSSION TITLE
Remove "Character" Prompting

### DISCUSSION THREAD

**Discussion by thedotmack** [spdustin/ChatGPT-AutoExpert#61] [Ideas]
I think it might be a good idea to remove the character prompting from the python script.

Why add in more confusion and instruction to the model initially when it would be better thinking about code instead of creatively outputting instructions for the user.

I love the characters, they're great, I'm in to pop culture, it's fun... but this thing isn't saving files consistently. It isn't doing it's main task well... therefore it would serve to maybe de-confuse it that way.

Confused? me too lol.

**Comment by spdustin** [comment]
You're not wrong. It was a novelty thing when the whole concept was a novelty, and now it annoys even me. 😂 It's gone on the next version.

**Comment by thedotmack** [comment]
Not only that but it performs so much better without all the extra anything. Have to save on token size

AN
